Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/SwipeView.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
+{
+	using FormsElement = Forms.SwipeView;
+
+	public enum SwipeTransitionMode
+	{
+		Drag = -1,
```

Comment:
What's the rationale for starting at -1?

Reply:
It is not necessary, changed.